support-frontend/assets/components/paymentMethodSelector/paymentMethodSelectorAccordionRow.tsx

```diff
@@ -96,53 +94,45 @@ export function ExistingPaymentMethodAccordionRow({
 	paymentMethod,
 	preExistingPaymentMethod,
 	existingPaymentMethod,
-	updateExistingPaymentMethod,
+	onChange,
 	expanded,
 	accordionBody,
 	checked,
 }: ExistingPaymentMethodAccordionRowProps): EmotionJSX.Element {
+	const image =
+		preExistingPaymentMethod.paymentType === 'Card' ? (
+			<SvgCreditCard />
+		) : (
+			<SvgDirectDebit />
+		);
+
 	return (
 		<div css={checked ? focused : notFocused}>
-			<div>
-				<Radio
+			<div css={[...(checked && accordionBody ? [borderBottom] : [])]}>
+				<RadioWithImage
 					id={`paymentMethod-existing${preExistingPaymentMethod.billingAccountId}`}
 					name="paymentMethod"
-					type="radio"
-					value={preExistingPaymentMethod.paymentType}
-					onChange={() => {
-						updateExistingPaymentMethod?.(preExistingPaymentMethod);
-					}}
+					onChange={onChange}
 					checked={
 						paymentMethod ===
 							mapExistingPaymentMethodToPaymentMethod(
 								preExistingPaymentMethod,
 							) && existingPaymentMethod === preExistingPaymentMethod
 					}
-					aria-labelledby="payment_method"
-					aria-expanded={expanded}
-					label={
-						<PaymentMethodLabel
-							label={getExistingPaymentMethodLabel(preExistingPaymentMethod)}
-							logo={
-								paymentMethodData[
-									mapExistingPaymentMethodToPaymentMethod(
-										preExistingPaymentMethod,
-									)
-								].icon
-							}
-							isChecked={existingPaymentMethod === preExistingPaymentMethod}
-						/>
-					}
-				/>
-
-				<div css={explainerListContainer}>
-					Used for your{' '}
-					{subscriptionsToExplainerList(
+					cssOverrides={[
+						radio,
+						...(checked && accordionBody ? [borderBottom] : []),
+						existingPaymentMethodOverrides,
+					]}
+					isSupporterPlus={true}
```

Comment:
I can't quite see what this is doing, is it right that it's hardcoded?

Reply:
Seems to affect something about the component styling, I'm not entirely sure why it's there either 🤷